src/actions/fetch-data-resource.js

```diff
@@ -0,0 +1,43 @@
+import axios from 'axios'
+
+export async function fetchResource (id) {
+
+  const instance = axios.create({
+    transformRequest: [
+      (data, headers) => {
+        headers.common.Accept = 'application/sparql-results+json'
+        return data
+      },
+    ]
+  })
+
+  const query =
+    `
+    DESCRIBE <http://qa.stad.gent/id${id}> <https://qa.stad.gent/id${id}>
+    `
+
+  let formData = new FormData()
+  formData.set('query', query)
+
+  let response
+
+  try {
+    response = await instance({
+      method: 'post',
+      url: 'https://qa.stad.gent/sparql',
+      data: formData,
+    })
+  }
+  catch (e) {
+    throw e
```

Comment:
On second thoughts: this is kind of useless. Catching an error provides you with the possibility of handling the error in a custom way. Just throwing that same error again has the same outcome as just leaving out the `try...catch` statement